keyboards/thevankeyboards/minivan/keymaps/dcompact/config.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2020 Dan Herrera
```

Comment:
This file can be removed, if there is no intention to add any config overrides.

Reply:
Done 👍